lean4/README.md

```diff
@@ -7,7 +7,11 @@ import Mathlib
 abbrev putnam_easy_solution : ℕ := sorry
 -- 2
 
+/-- What is 1 + 1 -/
 theorem putnam_easy : 1 + 1 = putnam_easy_solution := sorry
 ```
 
 To generate versions with the solution inlined into the question, you can use `python scripts/rewrite_solutions.py`.
+
+The docstring on the theorem must stay in sync with `informal/putnam.json`,
```

Comment:
We can mention that it is checked by CI and out-of-sync comments will not let the PR get merged.

Reply:
I've added a sentence about this being checking in CI.

I have not added anything saying "CI has to pass for PRs to be merged" because this is like writing "You should read the README" or similar - it's implied in the act of contributing to anything on github.

If you really feel strongly about this mention, you can tick this box in the repo settings:

![image](https://github.com/user-attachments/assets/81533d7f-bd4f-4c8c-a282-ec81bff6a0c3)


which will make the github PR page say "this can't be merged because CI fails" when CI fails.

Reply:
It is okay. I wanted to make sure that people at least know this is checked, which was not mentioned earlier.